internal/internal_task_handlers.go

```diff
@@ -148,6 +148,10 @@ type (
 		currentIndex  int
 		next          []*s.HistoryEvent
 	}
+
+	localActivityTimedOutError struct {
```

Comment:
I find this name confusing. It looks like it indicates that a local activity timed out and caused this error. But this indicates the forced decision completion.

Reply:
I'll call this decision heartbeat error.